extensions/amp-iframe/amp-iframe.md

```diff
@@ -208,3 +208,13 @@ Iframes are identified as tracking/analytics iframes if they appear to serve no
 ## Validation
 
 See [amp-iframe rules](https://github.com/ampproject/amphtml/blob/master/extensions/amp-iframe/0.1/validator-amp-iframe.protoascii) in the AMP validator specification.
+
+## Guideline: prefer existing AMP components to `amp-iframe`
+
+`amp-iframe` should be considered a fallback if the required user experience is not possible by other means in AMP. This is because there are many benefits to using a component tailored for a specific use-case instead, such as
+
+- Better resource management and performance
+- Increased loading priority (`amp-iframe` loads last, because AMP doesn't know what's inside of it)
```

Comment:
Got it—is there anything we can say around better loading priority for specific components v. amp-iframe, or should this line just be deleted?